test.js

```diff
@@ -2,3 +2,4 @@ require('./index.js');
 const assert = require('assert');
 
 assert.equal(true, tru);
+assert.equal(true, fals);
```

Comment:
false lolol

Reply:
yo i swear today has been the longest day

Reply:
yeah my bad.. highest effort shitpost💀